frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/MyGitBranchCheckoutOperation.kt

```diff
@@ -0,0 +1,16 @@
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
```

Comment:
https://github.com/JouriM66/KotlinMessages

Reply:
Wow this repo looks... obscure/esoteric